README.rst

```diff
@@ -16,7 +16,29 @@ Installation
 
 To install before an official pypa release::
 
-   pip install git+https://github.com/Princeton-CDH/django-parasol/tree/develop#egg=django-parasol
+   pip install git+https://github.com/Princeton-CDH/django-parasol@develop#egg=parasol
+
+
+To use with Django:
+
+    * Add `parasol` to **INSTALLED_APPS*
+    * Configure **SOLR_CONNECTIONS** in your django settings::
+
+        SOLR_CONNECTIONS = {
+            'default': {
+            'URL': 'http://localhost:8983/solr/',
+            'COLLECTION': 'name',
+            }
+        }
+
+    * Define a `SolrSchema` with fields and field types for your project.
+    * Run ``solr_schema`` manage command to configure your schema; it will
+      prompt to create the Solr core if it does not exist.
+
+      .. Note::
```

Comment:
Do you mean a django config to explicitly configure the schema object? I wondered about that, but it seems like more work (have to parse the string and load the class that way) without an obvious benefit.

Reply:
That was exactly it, and also the balance I wasn't sure about, because it does add some complication. If could see a situation where for some reason someone didn't want to import the schema, I'd say we need it at some point, but I can't come up with one.

Reply:
It just has to be imported somewhere within the django app so that it is identifiable somewhere, which doesn't seem terrible.... But something we can keep an eye on and adjust later if it becomes an annoyance.